crates/wit-parser/src/lib.rs

```diff
@@ -246,16 +262,19 @@ pub struct World {
     pub exports: IndexMap<WorldKey, WorldItem>,
 
     /// The package that owns this world.
+    #[serde(serialize_with = "serialize_optional_id")]
     pub package: Option<PackageId>,
 
     /// All the included worlds from this world. Empty if this is fully resolved
+    #[serde(skip)]
     pub includes: Vec<WorldId>,
 
     /// All the included worlds names. Empty if this is fully resolved
+    #[serde(skip)]
     pub include_names: Vec<Vec<IncludeName>>,
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize)]
```

Comment:
Does this need `Serialize` if it's `serde(skip)`'d above?

Reply:
Done.